src/master-playlist-controller.js

```diff
@@ -1348,9 +1351,11 @@ export class MasterPlaylistController extends videojs.EventTarget {
       if (variant.excludeUntil === Infinity) {
         return;
       }
-      // TODO: should we really assume codecs?
+      // TODO: should we really assume codecs
```

Comment:
It may be worth expanding on this comment. Maybe something like.

```
TODO decide whether two codecs should be assumed
Right now, for playlists that don't specify codecs, VHS assumes that there are two (one for audio and one for video). Although this is often the case, this may lead to broken behavior if the playlist only has one codec. It may be better in the future to decide at time of segment download how many tracks there are and determine the proper codecs. This will come at a cost of potentially more bandwidth, but will be a more robust approach than the assumption here.